lib/src/base-styles/_nx-forms.scss

```diff
@@ -47,6 +47,11 @@
   }
 }
 
+// hack to get Safari to recognize the legend's text margins
+.nx-legend {
+  -webkit-margin-collapse: separate;
```

Comment:
Are there any side-effects to this or does it just seem to do what we want here?

Reply:
> Are there any side-effects to this or does it just seem to do what we want here?

I checked in other browsers and couldn't find any side effects.

Reply:
> Are there any side-effects to this or does it just seem to do what we want here?

I couldn't find any side effects